imports/components/player_list.jsx

```diff
@@ -0,0 +1,44 @@
+export class PlayerList extends React.Component {
+  render() {
+    var players = this.props.players.map(function(player) {
+      return (<PlayerListEntry key={player._id} {...player} />);
+    });
+    var logIn = null;
+    if (!this.props.loggedIn) {
+      logIn = <li>
+        <strong>Log In</strong> to join this list
+      </li>
```

Comment:
You probably want to start with this in your eslintrc if you don't have it already:

```
{
    "extends": [
        "airbnb",
        "plugin:meteor/recommended",
    ],
    "plugins": [
        "meteor",
        "react",
        "jsx-a11y",
        "import",
    ],
    "settings": {
        "import/resolver": "meteor",
    }
}
```

(You may be missing the import resolver)

Here are the rules we used for jolly-roger, which may or may not be what you want:

```
    // explicit returns can sometimes be more readable and avoid the issue of "I want to return an
    // object literal but ES6 interprets it as a function body with a void expression"
    "arrow-body-style": ["off"],

    // Publish functions don't really need to be named.
    "func-names": ["off"],

    // For Match expressions
    "new-cap": ["error", { "capIsNewExceptions": ["Optional", "OneOf", "Maybe", "ObjectIncluding"] }],

    // sometimes else after a return reads more clearly
    "no-else-return": ["off"],

     // We sometimes use these in various forms
    "no-underscore-dangle": ["error",
      {
        "allow": [
          "_id", "_this", "_ensureIndex", "_dropIndex", "_redirectUri", "_postRequest",
          "_anyMethodsAreOutstanding", "_schemaKeys", "_schema", "_retrieveCredentialSecret",
        ],
      }
    ],

    // We totally use React.createClass and mixins all over; refactor this later
    "react/prefer-es6-class": ["off"],

    // We namespace all the React-Bootstrap stuff under "BS", which trips this rule
    "react/jsx-pascal-case": ["off"],

    // We have files with multiple React components.  Maybe we should split them out
    // later, but right now I don't want to deal with this
    "react/no-multi-comp": ["off"],

    // Specifying explicit propTypes is worthwhile
    "react/prefer-stateless-function": ["off"],

    // While this rule is generally useful, it currently gets confused by Meteor's import rewriting
    // for import { thing } from "meteor/meteorpackagename", so we need it to chill out.
    "import/no-extraneous-dependencies": ["off"],

    // I appear to have missed the memo on always preferring default exports to named ones.
    // I think I like named exports better, honestly.
    "import/prefer-default-export": ["off"],
```